### PR TITLE
Target Node.js 18, switch to `ky`

### PR DESCRIPTION
**Pull request by tommy-mitchell**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ console.log(await npmUser('sindresorhus'));
 {
 	name: 'Sindre Sorhus',
 	avatar: 'https://gravatar.com/avatar/d36a92237c75c5337c17b60d90686bf9?size=496',
```
Comment (tommy-mitchell) [inline readme.md]: This now returns `/npm-avatar/…`, which is only valid when prefixed with `https://www.npmjs.com`. Should that added to the result?

Comment (sindresorhus) [reply readme.md]: It should be an absolute URL, yes.

File: index.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import got from 'got';
-import cheerio from 'cheerio';
+import ky from 'ky';
+import {load} from 'cheerio';
```
Comment (sindresorhus) [inline index.js]: `load` is not a useful name without context. Rename it to something better.

**Comment by tommy-mitchell** [comment]
Updated.

**Pull request merged**